Source/BugsnagConfiguration.m

```diff
@@ -53,6 +54,15 @@ @interface BugsnagConfiguration ()
 
 @implementation BugsnagConfiguration
 
++ (instancetype _Nullable)loadConfig {
```

Comment:
Throwing exceptions are not a standard control flow mechanism in Cocoa - the program can continue running without this method succeeding. However, it could return more info about what failed, so I'll update it to do that.

> In many environments, use of exceptions is fairly commonplace. For example, you might throw an exception to signal that a routine could not execute normally—such as when a file is missing or data could not be parsed correctly. Exceptions are resource-intensive in Objective-C. You should not use exceptions for general flow-control, or simply to signify errors.

[Documentation Archive - Exception Handling](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ObjectiveC/Chapters/ocExceptionHandling.html)